### DISCUSSION TITLE
web development

### DISCUSSION THREAD

**Discussion by Saeed-280** [community/community#33107] [Other Feature Feedback, Questions, & Ideas]
Hello people,
I have started to learn the full= stack web development
so far i have completed the CSS and about t0 start the front end frame work {bootsrarp}

problem: I have published a website on **GitHub pages** ,CSs is not being applied 

links:
https://github.com/smdsayeed/Cv    for- repo
https://smdsayeed.github.io/Cv/       for-webpage

<link rel="stylesheet" href="https://github.com/smdsayeed/Cv/blob/main/index.css">   code which i have written to link the css

**Comment by radar-blip** [comment]
The link in your index.html to your .css is wrong.

 Change from:  `<link rel="stylesheet" href="https://github.com/smdsayeed/Cv/blob/main/index.css">`
 To:   `<link rel="stylesheet" href="index.css">`

**Comment by Saeed-280** [comment]
What it should be?

**Reply by radar-blip** [reply]
Change from: `<link rel="stylesheet" href="https://github.com/smdsayeed/Cv/blob/main/index.css">`
To: `<link rel="stylesheet" href="index.css">`

**Comment by Saeed-280** [comment]
YYay!
It worked ...!
Thanks you so much man

**Reply by radar-blip** [reply]
You're welcome. Glad I could help! 😃

**Comment by AustingGerald** [comment]
As for me, I prefer resolve problems instead of leave them as they are. I always turn to this website https://evacodes.com/ which, as far as I can see, may also help you to resolve all the problems with software issues for your computer for example. I really like these specialists and I like turning to them to get helped.

**Comment by DrakeYounger** [comment]
I took a look at your GitHub repository and noticed that your CSS file is located in the ""CSS"" folder. Therefore, you need to change your link to the CSS file to this:
<link rel=""stylesheet"" type=""text/css"" href=""CSS/style.css"">. 

Also, make sure that the folder name is exactly the same as it appears in your link. You can also try using [Corona web design](https://roistrategicmarketing.com/web-design/corona/). I think it will be useful for you to hear different opinions on how to solve this issue. As for your website, I visited it and it seems that your CSS file is not being applied. This could be due to a couple of reasons. Firstly, ensure that the link to your CSS file is correct and that the file path is accurate. Secondly, make sure that you've saved the changes to your CSS file before pushing it to GitHub.

**Comment by Xelliese** [comment]
Great job on learning full-stack web development! Your issue with CSS not showing on GitHub Pages might be caused by the wrong link to your CSS file. Check if the path you've written matches the one in which your CSS file is in your project. Sometimes, you might use the wrong slash or misspell the file name.

Here's a simple example: If you have a folder named styles with your CSS, your link should look like this:
`<link rel="stylesheet" href="styles/style.css">`

Also, as you keep learning, you should use a <a href="https://pbb.lviv.ua/en/">site optimization service</a> later. These services can help fix issues like this and give tips on making your website load faster and work better.
